recipes/ogdf/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def validate(self):
+        if is_msvc(self) and self.options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} can not be built as shared on Visual Studio and msvc.")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def requirements(self):
+        self.requires("coin-clp/1.17.7")
+        self.requires("pugixml/1.13")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
+        tc.variables["COIN_SOLVER"] = "CLP"
+        tc.variables["COIN_SOLVER_IS_EXTERNAL"] = 0
+        tc.generate()
+        tc = CMakeDeps(self)
+        tc.generate()
+
+    def _patch_sources(self):
+        # delete code from other cci packages
+        rmdir(self, join(self.source_folder, "src", "coin"))
+        rmdir(self, join(self.source_folder, "include", "coin"))
+        rmdir(self, join(self.source_folder, "include", "ogdf", "lib", "backward"))
+        rmdir(self, join(self.source_folder, "src", "ogdf", "lib", "pugixml"))
+        rmdir(self, join(self.source_folder, "include", "ogdf", "lib", "pugixml"))
+        # use cci packages where available
+        replace_in_file(self, join(self.source_folder, "CMakeLists.txt"), "include(coin)", "find_package(coin-clp REQUIRED CONFIG)\nfind_package(pugixml REQUIRED CONFIG)")
+        replace_in_file(self, join(self.source_folder, "cmake", "ogdf.cmake"), "target_link_libraries(OGDF PUBLIC COIN)", "target_link_libraries(OGDF PUBLIC coin-clp::coin-clp pugixml::pugixml)")
+        # replace pugixml copy in repo by conan dependency
+        for dir_name, file_name in [("include", "GexfParser.h"),
+                                    ("include", "GraphMLParser.h"),
+                                    ("include", "SvgPrinter.h"),
+                                    ("include", "TsplibXmlParser.h"),
+                                    ("src", "GraphIO_graphml.cpp"),
+                                    ("src", "GraphIO_gexf.cpp")]:
+            replace_in_file(self, join(self.source_folder, dir_name, "ogdf", "fileformats", file_name), "ogdf/lib/pugixml/pugixml.h", "pugixml.hpp")
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build(target="OGDF")
+
+    def package(self):
+        copy(self, pattern="LICENSE.txt", src=self.source_folder, dst=join(self.package_folder, "licenses"))
```

Comment:
```suggestion
        copy(self, pattern="LICENSE*.txt", src=self.source_folder, dst=join(self.package_folder, "licenses"))
```
So you copy all license files to the licenses folder